docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - BOPTEST_REDIS_HOST
       - BOPTEST_REGION
       - BOPTEST_TIMEOUT
+      - BOPTEST_MESSAGE_TIMEOUT
```

Comment:
On the other hand. BOPTEST_MESSAGE_TIMEOUT is how long the web server waits for a response from the worker running a test.